portals/devportal/source/src/app/components/Apis/Details/ApiConsole/TryOutController.jsx

```diff
@@ -574,332 +587,446 @@ function TryOutController(props) {
+                                                    defaultMessage={
+                                                        'You can use your existing Access Token or '
+                                                                + 'you can generate a new Test Key.'
+                                                    }
                                                 />
                                             )}
-                                            name='password'
-                                            onChange={handleChanges}
-                                            type={showPassword ? 'text' : 'password'}
-                                            value={password || ''}
-                                            fullWidth
-                                            InputProps={{
-                                                autoComplete: 'new-password',
-                                                endAdornment: (
-                                                    <InputAdornment position='end'>
-                                                        <IconButton
-                                                            edge='end'
-                                                            aria-label='toggle password visibility'
-                                                            onClick={() => setShowPassword(!showPassword)}
-                                                        >
-                                                            {showPassword ? <Visibility /> : <VisibilityOff />}
-                                                        </IconButton>
-                                                    </InputAdornment>
-                                                ),
-                                            }}
-                                        />
-                                    </Grid>
-                                </>
-                            )}
-
-                            {securitySchemeType !== 'BASIC' && securitySchemeType !== 'TEST' && (
+                                        >
+                                            <Box m={1} mt={2}>
+                                                <IconButton
+                                                    aria-label='Use existing Access Token or generate a new Test Key'
+                                                >
+                                                    <HelpOutline />
+                                                </IconButton>
+                                            </Box>
+                                        </Tooltip>
+                                    </>
+                                )}
+                            </Grid>
+                        </Box>
+                    ) : (
+                        <Box display='block' justifyContent='center' className={classes.authHeader}>
```

Comment:
Shall we move this advertise info related logic to a seperate component and use here

Reply:
Done with https://github.com/wso2/apim-apps/pull/130/commits/5aac85c885fae9e259de37df89f53c6fc9b5d5dd